### ISSUE TITLE
MPV *not* displaying updated tags for particularly wrapped lossless streams from icecast (ogg container) 

### ISSUE THREAD

**Issue by Parakovsky** [mpv-player/mpv#15063]
### mpv Information

```bash
mpv v0.39.0 Copyright © 2000-2024 mpv/MPlayer/mplayer2 projects
 built on Sep 23 2024 17:46:22
libplacebo version: v7.349.0
FFmpeg version: n7.0.2
FFmpeg library versions:
   libavcodec      61.3.100
   libavdevice     61.1.100
   libavfilter     10.1.100
   libavformat     61.1.100
   libavutil       59.8.100
   libswresample   5.1.100
   libswscale      8.1.100
```

### Other Information

```text
- Linux version: ARCH
- Kernel Version: 6.11
- GPU Model:
- Mesa/GPU Driver Version:
- Window Manager and Version: sway
- Source mpv:
- Introduced in version:
```

### Reproduction Steps

If performed playback on parakovsky.duckdns.org:8030/liq.ogg only displays the initial track name but not the ones that comes after. 

### Expected Behavior

If performed on another stream with the same container but with lossy compression method (ogg 320 kbps) metadata tags on track changes displays correctly test link parakovsky.duckdns.org:8030/lossy.ogg

### Actual Behavior

After the initial track title displayed of the track playing at the moment of the stream mpv will keep displaying only one name as if it is continious even tho the track could be already changed. Was curious and talked with Icecast people who tested the stream and stuff and verified that the back frame configuration correct and trough ogginfo everything looks Gucci. So, having this ticket is the last resort for a poor operator to test the problem and to keep track of it. 

### Log File

[output.txt](https://github.com/user-attachments/files/17348912/output.txt)

### I carefully read all instruction and confirm that I did the following:

- [X] I tested with the latest mpv version to validate that the issue is not already fixed.
- [X] I provided all required information including system and mpv version.
- [X] I produced the log file with the exact same set of files, parameters, and conditions used in "Reproduction Steps", with the addition of `--log-file=output.txt`.
- [X] I produced the log file while the behaviors described in "Actual Behavior" were actively observed.
- [X] I attached the full, untruncated log file.
- [X] I attached the backtrace in the case of a crash.

**Comment by llyyr** [comment]
I don't see the metadata update at all here, even your Icecast Dashboard doesn't have metadata for the lossless stream.
![image](https://github.com/user-attachments/assets/ca06ac84-e971-426a-8960-a776ecbee1b5)

**Comment by llyyr** [comment]
Actually, this is a ffmpeg issue. It's fixed by https://ffmpeg.org//pipermail/ffmpeg-devel/2023-April/308780.html

**Comment by Parakovsky** [comment]
YES! and to your point sometimes the library would play some sourced
from youtube track or lekchur that don't have a metadata so..... it
probably wouldn't display itself. and yes. i have ffmpeg 6.1 on the
liquidsoap i don't think it works on ffmpeg 7 so far. wonder if my
version is older than the patch and could i have been patching it
independently somehow. thanks for your feedback. ps: foobar seems to
display metadata correctly which probably means the patch is already
there but not helping our beloved m.p.v to work it out. just saying.

**Comment by raszpl** [comment]
maybe related https://github.com/smplayer-dev/smplayer/issues/1346

37:

```
mpv http://s3.viastreaming.net:8835/listen.pls  --display-tags=*
Playing: http://s3.viastreaming.net:8835/;
 (+) Audio --aid=1 (mp3 2ch 44100Hz)
File tags:
 icy-notice1: <BR>This stream requires <a href="http://www.winamp.com">Winamp</a><BR>
 icy-notice2: Shoutcast DNAS/posix(linux x64) v2.6.1.777<BR>
 icy-name: Dark Ambient Radio - Muzak for the Final Elevation
 icy-genre: Ambient
 icy-br: 192
 icy-sr: 44100
 icy-url: https://www.darkambientradio.de
 icy-pub: 0
AO: [wasapi] 48000Hz stereo 2ch float
File tags:
 icy-title: Phallus Dei - Queen-Bee (Hides Amongst The Dead)
A: 00:00:02 / 00:00:13 (21%) Cache: 10s/457KB^C
```

41:

```
>mpv http://s3.viastreaming.net:8835/listen.pls  --display-tags=*
Playing: http://s3.viastreaming.net:8835/;
 ● Audio  --aid=1  (mp3 2ch 44100 Hz 192 kbps)
AO: [wasapi] 48000Hz stereo 2ch float
A: 00:00:02 / 00:00:13 (22%) Cache: 10s/455KB^C
```
